src/OrleansServiceBus/Providers/Streams/EventHub/EventHubConstants.cs

```diff
@@ -0,0 +1,12 @@
+namespace Orleans.ServiceBus.Providers
+{
+    internal class EventHubConstants
+    {
+        public readonly static string StartOfStream =
+#if NETSTANDARD
+            Microsoft.Azure.EventHubs.PartitionReceiver.StartOfStream;
+#else
+            Microsoft.ServiceBus.Messaging.EventHubConsumerGroup.StartOfStream;
+#endif
+    }
```

Comment:
It is used on 2 files... I think have this constant is OK in a separated class. However, I'll make it internal.